pontoon/settings/base.py

```diff
@@ -728,6 +728,18 @@ def _allowed_hosts():
     'vi-vn', 'wo-sn', 'xh-za', 'yo-ng', 'zh-cn', 'zh-hk', 'zh-tw', 'zu-za',
 ]
 
+# Bleach tags and attributes
+ALLOWED_TAGS = [
+    'a', 'abbr', 'acronym', 'b', 'blockquote', 'br', 'code', 'em', 'i',
+    'li', 'ol', 'p', 'strong', 'ul',
+]
```

Comment:
I wanted to have better control for allowed tags and attributes as we will use same for other inputs (project info for ex.). It will also help to avoid changes in bleach libraries defaults. Also having complete list of all tags and attributes in settings makes it easy to control them.